SPECS/msft-golang/msft-golang.spec

```diff
@@ -1,3 +1,15 @@
+Summary:        Go
+Name:           msft-golang
+Version:        1.19.12
+Release:        1%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Security
+URL:            https://github.com/microsoft/go
+Source0:        https://github.com/microsoft/go/releases/download/v1.19.12-1/go.20230802.5.src.tar.gz
+Source1:        https://dl.google.com/go/go1.4-bootstrap-20171003.tar.gz
+Patch0:         go14_bootstrap_aarch64.patch
 %global goroot          %{_libdir}/golang
```

Comment:
I think even though the linter yells at us for these macros to be placed here, our convention generally has been to keep all the macros at the top of the spec file.

Reply:
it seemed like the linter yelled about them being at the top.  i put them where the linter looked like it wanted.  i'll look to see if i can run the linter locally.

Reply:
The linter expects us to keep them where they currently are.
But for some reason (convention or whatever - I am not sure) we tend to keep them at the top.
/cc @PawelWMS

Reply:
sorry, i misread your message.  i will move the macros back to the top.

Reply:
moved macros back to the top